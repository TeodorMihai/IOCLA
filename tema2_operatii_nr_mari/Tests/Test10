-FAFAFA12FCAFEBABE >> 10
