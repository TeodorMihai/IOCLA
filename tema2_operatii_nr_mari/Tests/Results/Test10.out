BEBAFECA2FA1AFAF0F
10
FECA2FA1AFAF0F
-FAFAFA12FCAFE
